Les fichiers binaires fusiondirectory.pre2/include/.class_plugin.inc.swp et fusiondirectory/include/.class_plugin.inc.swp sont différents
Les fichiers binaires fusiondirectory.pre2/include/simpleplugin/.class_simplePlugin.inc.swp et fusiondirectory/include/simpleplugin/.class_simplePlugin.inc.swp sont différents
diff -Nur fusiondirectory.pre2/plugins/personal/roles/class_userRoles.inc fusiondirectory/plugins/personal/roles/class_userRoles.inc
--- fusiondirectory.pre2/plugins/personal/roles/class_userRoles.inc	2015-09-23 21:17:50.000000000 +0200
+++ fusiondirectory/plugins/personal/roles/class_userRoles.inc	2015-10-10 15:28:55.912173038 +0200
@@ -74,7 +74,17 @@
     $this->ignore_account = FALSE;
 
     /* Groups handling */
-    $groups    = objects::ls('ogroup');
+    $groups      = array();
+    $groupsattrs = objects::ls('ogroup',
+                                array('cn' => 'raw',  'description' => 'raw'));
+    foreach($groupsattrs as $dn => $groupattr) {
+        if(isset($groupattr['description'][0])) {
+            $groupDisplay = $groupattr['description'][0]." [".$groupattr["cn"][0]."]";
+        } else {
+            $groupDisplay = $groupattr['cn'][0];
+        }
+        $groups[$dn] = $groupDisplay;
+    }
     $this->attributesAccess['groupsMembership']->setInLdap(FALSE);
     $this->attributesAccess['groupsMembership']->attribute->setChoices(array_keys($groups), array_values($groups));
     if ($this->is_template) {
Les fichiers binaires fusiondirectory.pre2/plugins/personal/roles/.class_userRoles.inc.swp et fusiondirectory/plugins/personal/roles/.class_userRoles.inc.swp sont différents
